teslausb-www/teslausbsecure.nginx

```diff
@@ -0,0 +1,29 @@
+server {
+    listen 80 default_server;
+    listen [::]:80 default_server;
+
+    root /var/www/html;
+
+    index index.html
+
+    server_name _;
+
+    location / {
+        try_files $uri $uri/ =404;
+        auth_basic "Restricted Content";
```

Comment:
I don't like having this be a separate .nginx file for this (which then needs to be kept in sync with the other one).
Can this be done with an `include` directive so that only this part needs to be separate between the auth/no-auth cases? Or alternatively during setup replace the "Restricted Content" string with "off" if the user didn't specify the auth variables.

Reply:
Good idea, I went with the latter suggestion, thank you.